components/channel_header/channel_header.js

```diff
@@ -447,11 +445,7 @@ class ChannelHeader extends React.PureComponent {
                     popoverSize='lg'
                     style={{maxWidth: `${this.state.popoverOverlayWidth}px`, transform: `translate(${this.state.leftOffset}px, ${this.state.topOffset}px)`}}
                     placement='bottom'
-                    className={classNames(['channel-header__popover',
-                        {
-                            'chanel-header__popover--lhs_offset': this.props.hasMoreThanOneTeam,
-                            'chanel-header__popover--new_sidebar': !isLegacySidebar,
-                        }])}
+                    className={classNames('channel-header__popover', {'chanel-header__popover--lhs_offset': this.props.hasMoreThanOneTeam})}
```

Comment:
The different margins with the new sidebar have been rolled into the base class.